### PR TITLE
First Ecosystem Spotlight blog post: tf-controller

### PR DESCRIPTION
**Pull request by dholbach**
cc @priyanka-ravi

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-1148--fluxcd.netlify.app/blog/

File: content/en/blog/2022-09-14-how-to-gitops-your-terraform/index.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+   readme](https://github.com/weaveworks/tf-controller/tree/main/charts/tf-controller#tf-controller-for-flux).
+
+1. Alternatively, you can install Terraform Controller via kubectl:
+
+   ```cli
+   export TF_CON_VER=v0.12.0
+   kubectl apply -f https://github.com/weaveworks/tf-controller/releases/download/${TF_CON_VER}/tf-controller.crds.yaml
+   kubectl apply -f https://github.com/weaveworks/tf-controller/releases/download/${TF_CON_VER}/tf-controller.rbac.yaml
+   kubectl apply -f https://github.com/weaveworks/tf-controller/releases/download/${TF_CON_VER}/tf-controller.deployment.yaml
+   ```
```
[the remark this answers is not in the document]
Comment (dholbach) [reply content/en/blog/2022-09-14-how-to-gitops-your-terraform/index.md:116]: Ok, I can replace it with a link to the docs.

Can you elaborate on what you would like to see in the flux bootstrap repo? I'm not sure I understand.

[the remark this answers is not in the document]
Comment (dholbach) [reply content/en/blog/2022-09-14-how-to-gitops-your-terraform/index.md]: Ok, got it!

**Review by stefanprodan — APPROVED** [review]
LGTM

**Pull request merged**